### PR TITLE
using blazegraph-runner to load and reason gaf models into blazegraph

### PR DESCRIPTION
**Pull request by dougli1sqrd**

### CODE REVIEW

File: pipeline/Makefile [hunk 1]
```diff
@@ -145,10 +142,15 @@ BG = java -server -XX:+UseG1GC -Xmx$(BGMEM) -cp $(BGJAR) com.bigdata.rdf.store.D
 # Load blazegraph
 
 BG_PORT = 8899
```
Comment (kltm) [inline pipeline/Makefile]: We should either have found clear port, or at least randomized port, used here.

Comment (dougli1sqrd) [inline pipeline/Makefile]: That's true. For the moment though this isn't being actively used. Once we have sparta checking integrated though, we should definitely think about that.

Comment (kltm) [inline pipeline/Makefile]: Okay, talked to @dougli1sqrd , all good

**Comment by dougli1sqrd** [comment]
Something's wrong with the makefile, let's not merge yet.

**Comment by dougli1sqrd** [comment]
Everything is fixed: https://build.berkeleybop.org/job/go-gaf-release-experimental/146/

**Pull request merged**